substratevm/mx.substratevm/mx_substratevm.py

```diff
@@ -327,6 +327,7 @@ def __init__(self, image_deps=None, builder_deps=None, native_deps=None):
     'native-image' : ToolDescriptor(),
     'junit' : ToolDescriptor(builder_deps=['mx:JUNIT_TOOL', 'JUNIT', 'HAMCREST']),
     'regex' : ToolDescriptor(image_deps=['regex:TREGEX']),
+    'jmh' : ToolDescriptor(),
```

Comment:
Ideally we would not need this and instead annotate the two features with `@AutomaticFeature` and use a shared `com.oracle.svm.hosted.ServiceLoaderFeature#isInConfiguration` implementation to decide if the features should be active or not. @christianwimmer do you agree?

Reply:
A good example for auto enabling is com.oracle.svm.polyglot.kotlin.KotlinFeature